pkg/service/manipulator_test.go

```diff
@@ -19,6 +19,36 @@ func TestNewManipulator(t *testing.T) {
 	assert.NotNil(t, m)
 }
 
+// Integration test to verify the flow of WebP image is requested without having support of WebP on client's side
```

Comment:
I've got a question here, how is this being determined? Based on the `Accept` header from the client or something else?

Reply:
Yes, based on the Accept header from the caller. The handler should put the list of them as an array of strings and pass them as the formats when constructing the Spec using SpecBuilder. You can inspect the network calls chrome or webview make when downloading an image and compare to non webP supported browsers.